compiler-plugin/src/main/scala/scalapb/package.scala

```diff
@@ -1,11 +1,13 @@
+import com.trueaccord.scalapb.compiler.ProtobufGenerator
 import protocbridge.JvmGenerator
 
 package object scalapb {
   def gen(
     flatPackage: Boolean = false,
     javaConversions: Boolean = false,
     grpc: Boolean = true,
-    singleLineToString: Boolean = false): (JvmGenerator, Seq[String]) =
+    singleLineToString: Boolean = false,
+    collectionType: String = "scala.collection.Seq"): (JvmGenerator, Seq[String]) =
```

Comment:
A field level set is definitely a good use case but for our projects being able to set a global collection type for repeated items different than `Seq` would be really helpful. `Seq` isnt very strict so that the functional programming libs we uses don't support it.